README.adoc

```diff
@@ -195,13 +196,13 @@ If you are using a a fresh installation of Docker, your DOCKER_HOST should be un
 
 This feature is available for three configurations :
 
- * Java 8 + Guice + Cassandra + RabbitMQ + Swift + ElasticSearch
+ * Java 8 + Guice + Cassandra + RabbitMQ + Swift + ElasticSearch + Ldap
```

Comment:
we don't update `how to run james with ldap` in this section. See below products! Document for ldap is updated in `packaging` section.

Reply:
Follow other product with Ldap, I removed it in this file.